project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+schema_version: '0.3'
+os_version: 3.7.0
+os_sha: d5269793f1
+buildstock_directory: ../ # Relative to this file or absolute
+project_directory: project_national # Relative to buildstock_directory
+output_directory: national_upgrades
+weather_files_url: https://data.nrel.gov/system/files/156/Buildstock_TMY3_FIPS-1678817889.zip
+# weather_files_path: c:/OpenStudio/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: residential_quota
+  args:
+    n_datapoints: 5
```

Comment:
Is it really just 5?

Reply:
let's use 100 for the small run test. will be 550000 for full run. Do we need a medium run?

Reply:
Not planning for medium run

Reply:
okay